README.md

```diff
@@ -27,7 +27,8 @@ You need:
 ## How to contribute
 > First, read through and familiarize yourself with our [ValidMind style guide](https://docs.validmind.ai/about/style-guide.html).
 
-Our core docs are sourced in Quarto Markdown under [`site`](https://github.com/validmind/documentation/tree/main/site/). If you create new documentation, make sure to add it to the [`_quarto.yml`](https://github.com/validmind/documentation/blob/main/site/_quarto.yml) file.
+- Our core docs are sourced in Quarto Markdown under [`site/guide`](https://github.com/validmind/documentation/tree/main/site/guide). 
```

Comment:
```suggestion
- Our core user guides are sourced in Quarto Markdown under [`site/guide`](https://github.com/validmind/documentation/tree/main/site/guide). 
```

I know you mentioned that this path choice is deliberate but it's still confusing to me, as there are lots of other things I would consider 'core docs' that aren't in this path. For example, the getting-started topics to me qualify as core docs, as they enable you to start using our product. Can we settle on this compromise?